src/common/style/vanilla/icons.scss

```diff
@@ -0,0 +1,189 @@
+@import 'vanilla-framework/scss/settings';
+
+@mixin icon-size($size) {
```

Comment:
Do these styles need to be copied here?

Is there a plan to move them to vanilla framework, so we can use icons from there?
If no, maybe we should include dashboard theme and use it from there?